### ISSUE TITLE
eu.rssw.antlr.proparse.checks.IndentationCheck false positive or wrong line flagged

### ISSUE THREAD

**Issue by ccecvb** [Riverside-Software/sonar-openedge#1104]
line 8  is flagged
```
define variable W-WIN as handle no-undo.

on end-error of W-Win /* Whor Window */
or endkey of W-Win anywhere do:
    /* This case occurs when the user presses the "Esc" key.
     In a persistently run window, just ignore this.  If we did not, the
     application would exit. */
    if this-procedure:persistent then return no-apply.
end.
``` 

but line 4 must be changed to make rule happy, code below does not trigger the rule

```
define variable W-WIN as handle no-undo.

on end-error of W-Win /* Whor Window */
or endkey of W-Win anywhere
do:
    /* This case occurs when the user presses the "Esc" key.
     In a persistently run window, just ignore this.  If we did not, the
     application would exit. */
    if this-procedure:persistent then return no-apply.
end.
``` 

as far as I'm concerned, line 4 was ok before the change

**Comment by gquerret** [comment]
Fixed in 2.23.2 (same issue as #1082, forgot to close this issue at the same time)
